.github/workflows/docs.yml

```diff
@@ -40,20 +62,72 @@ jobs:
       with:
         tool: wasm-pack
 
-    - name: Build Rust documentation
-      run: cargo doc --all --no-deps
-
     - name: Setup bun
       uses: oven-sh/setup-bun@v2
       with:
         bun-version: latest
 
-    - name: Build TypeScript and Kotlin docs
+    - name: Make TS Docs
       run: |
         cd crypto-ffi
         cargo make docs-ts
+
+    - uses: actions/upload-artifact@v4.5.0
+      with:
+        name: tsdoc
+        path: target/doc
+        retention-days: 1
+        overwrite: true
+        include-hidden-files: true
+
+
+  ktdoc:
+    name: Kotlin
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+      with:
+        rustflags: ''
+        target: |
+          x86_64-unknown-linux-gnu
```

Comment:
I don't think specifying the target here is necessary, one only needs that for additional, non-system targets.

Reply:
My reasoning here is, we should avoid unnecessary things like this, where we know omitting them would do just as well. Contrast that with `rustflags: ''` where we actually want to change the value.

Reply:
Ok, adjusted it.